bindata/network/ovn-kubernetes/common/008-script-lib.yaml

```diff
@@ -589,6 +589,19 @@ data:
         ovn_v6_join_subnet_opt="--gateway-v6-join-subnet {{.V6JoinSubnet}}"
       fi
 
+{{- if not .OVNIPsecEnable }}
+      # flush out xfrm state and policy entries so that it cleans up
+      # entries created for ovs. Since it's cleaning up whole entries,
+      # it may cause slight interruption in case of external ipsec
+      # configuration.
+      echo "remove ovs ipsec state if present"
+      ip x s flush
+      ip x p flush
+      rm -f /etc/ipsec.d/openshift.conf || true
+      # if pluto is running, we need to restart it after the flush
+      chroot /proc/1/root ipsec restart || true
```

Comment:
@bengal I remember there used to be a NM issue around this, not sure if fixed?

Reply:
AFAIK, restarting ipsec while there are active connections created by NM-libreswan would break them. You need to reactivate such connections manually to restore connectivity.

Reply:
doesnt feels very production grade.
if pluto daemon restart, for any reason (error? crash? etc)
all the connections will die and wont even attempt to restart

Reply:
Right, in case of crashes that's a problem. Is this issue already filed somewhere? If not please do, so that we can plan how to improve this in NM-libreswan.